### ISSUE TITLE
Github Action failure notification emails blocked by gmail

### ISSUE THREAD

**Issue by abhinav92003** [DynamoRIO/dynamorio#4790]
We use Github Actions to send an email to dynamorio-devs when there's some CI failure on master merges. We use the `dawidd6/action-send-mail@v2` (https://github.com/dawidd6/action-send-mail) action for this.

This is failing frequently recently: last 6/10 CI failures weren't able to send the notification email.

This is because gmail thinks it is a suspicious sign-in attempt and blocks it. Despite marking it as a known login attempt multiple times, it continues to get blocked.

https://github.com/dawidd6/action-send-mail/issues/17 discusses using generated app password + two-step verification for gmail account. Maybe that'll help -- only way would be to try.

**Comment by abhinav92003** [comment]
Trying this out. There will be notification email failures between turning on two-step verification and completing the rest of the process.

**Comment by abhinav92003** [comment]
All steps are complete. To test, I can create a new PR, modify the workflow condition, and try to trigger a failure in some test to see if notifications are being sent. Stability information will become known as we see master merge failures over some time and their notification email statuses.

**Comment by abhinav92003** [comment]
> To test, I can create a new PR, modify the workflow condition, and try to trigger a failure in some test to see if notifications are being sent. 

Working as intended.

> Stability information will become known as we see master merge failures over some time and their notification email statuses.

Will keep the issue open for a few days, observe, update and then close/take further steps if needed.

**Comment by abhinav92003** [comment]
The last security notification I got from Gmail was on April 1, despite the fact that we had quite a few master merge CI failures recently due to #4800. So I think the workaround of using app passwords + two-step verification helped.
